certbot-apache/certbot_apache/configurator.py

```diff
@@ -269,8 +293,13 @@ def _check_aug_version(self):
 
     def get_parser(self):
         """Initializes the ApacheParser"""
+        # If user provided vhost_root value in command line, use it
+        if self.conf("vhost-root"):
+            vhostroot = self.conf("vhost-root")
+        else:
+            vhostroot = None
         return parser.ApacheParser(
-            self.aug, self.conf("server-root"), self.conf("vhost-root"),
+            self.aug, self.option("server_root"), vhostroot,
```

Comment:
nit: Won't just passing `self.conf("vhost-root")` here get the same behavior as the `if` statements above?

Reply:
The `server_root` is a required parameter to parser, while vhost root is not. This is changed in order to make constant / cli variable handling more streamlined for overrides and main class alike. If we ever need to add functionality around setting or determining `server_root`, this allows it to be done in `ApacheConfigurator._prepare_options()`. For this reason I'd like to keep it as it is.

Reply:
I don't understand. What does the change I proposed have to do with `server_root`?

Reply:
Oh, I'm sorry, I misread the comment. You are absolutely correct here. This is now fixed.